Add pavd and daily LAI

angehung5 opened the pull request:
Update global canopy python script to get new daily files on AWS.
Update Table 2 in README by adding pavd and data time frequency.

angehung5 committed:
Update global_data_process.py

angehung5 committed:
Update Table 2

Add pavd and data time frequency

angehung5 committed:
Update Table 2

Add pavd and data time frequency

angehung5 committed:
Update global_data_process.py

angehung5 committed:
Update global_data_process.py

angehung5 committed:
Update global_data_process.py

angehung5 committed:
Update global_data_process.py

angehung5 committed:
Update global_data_process.py

drnimbusrain commented on README.md:
```diff
@@ -150,10 +150,11 @@ The Canopy-App input data in [Table 2](#table-2-canopy-app-required-input-variab
-| `ffrac`                          | Forest fraction (dimensionless)             | Based on [VIIRS GVF-NPP](https://www.star.nesdis.noaa.gov/jpss/gvf.php) and GriddingMachine/MODIS FFRAC/FVC from Terra.  Data Period=2020. ([DiMiceli et al., 2022](https://doi.org/10.5067/MODIS/MOD44B.061)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/p ...
+| `fh`                             | Forest canopy height (m)                    | Fused GEDI/Landsat data. Data Period=2020. Data frequency=Annual. ([Potapov et al., 2020](https://doi.org/10.1016/j.rse.2020.112165)) |
+| `clu`                            | Forest clumping index (dimensionless)       | GriddingMachine/MODIS. Data Period=2001-2017 Climatology. Data frequency=Monthly. ([Wei et al., 2019](https://doi.org/10.1016/j.rse.2019.111296)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB ...
+| `lai`                            | Leaf area index (m2/m2)                     | VIIRS-NPP. Data Period=2020. Data frquency=Daily, interpolated from original 8-day product. ([Myneni 2018](https://doi.org/10.5067/VIIRS/VNP15A2H.001)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzd ...
```
```suggestion
| `lai`                            | Leaf area index (m2/m2)                     | VIIRS-NPP. Data Period=2020. Data frequency=Daily, interpolated from original 8-day product. ([Myneni 2018](https://doi.org/10.5067/VIIRS/VNP15A2H.001)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
```

drnimbusrain reviewed (approved):
@angehung5 Thanks for the updates, well done!

I assumed you tested these updates to the global data process script and it gives expected daily LAI and new PAVD products.   Also seems like no updates are necessary for the python README for the updated script?

drnimbusrain committed:
Update README.md

angehung5 commented:
Yes, I have tested the updated script and it worked fine.
And no, we don't need to update the python README. I just changed the filenames (from YYMM01 to YYMMDD) and added pavd.
________________________________
From: Patrick Campbell ***@***.***>
Sent: Monday, October 16, 2023 8:36 PM
To: noaa-oar-arl/canopy-app ***@***.***>
Cc: Wei-Ting Hung ***@***.***>; Mention ***@***.***>
Subject: Re: [noaa-oar-arl/canopy-app] Add pavd and daily LAI (PR #95)


@drnimbusrain approved this pull request.

@angehung5<https://secure-web.cisco.com/1KtB5Ca4fK4SfrX9G0U3wLdcaDlA0Wpc9G2AHT00nvymptj6DK3DslQVUukvegna00rcTdI5oD6R9NEVnJ_r9UqXbUE9fHqx-75AwOEJEZ78gIDdXXwWTczifXjs4wvgpiRUBeGAsx_0SpDpUaXulx597fNHDq4WQN8U_G4Ol1jpt2PIiHRrnZ3yeE_0WeS0K_xBQ229RXrAInkG0FS5yfA19lcu6R2DBXOq2vJjXyMau9g10NYVVi7KDP7Yw5b7bxull7FRIPNm2Pl8iT_pVZq5a8nAo_vXKjQofKlTkTcdzo4_UcJZ3zQN9gDaKmLhRSZVWOrQMyCSBQ9rkm1YR8c37wpLPy_43wEtVq4ZhQ_k3lSEQkfil2wNbDJmqGDvpR7qH-Xh5aGxCTf3gbxBIdG-EGhJDzP2ctCniaAH270U/https%3A%2F%2Fgithub.com%2Fangehung5> Thanks for the updates, well done!

I assumed you tested these updates to the global data process script and it gives expected daily LAI and new PAVD products. Also seems like no updates are necessary for the python README for the updated script?

________________________________

In README.md<https://github.com/noaa-oar-arl/canopy-app/pull/95#discussion_r1361380625>:

> @@ -150,10 +150,11 @@ The Canopy-App input data in [Table 2](#table-2-canopy-app-required-input-variab
 | `hpbl`                           | Height of the planetary boundary layer (m)  | UFS NOAA/GFSv16                                    |
 | `prate_ave`                      | Average mass precipitation rate (kg m-2 s-1) | UFS NOAA/GFSv16                                   |
 | **External Canopy Variables**    | **Variable Description and Units**          | **Data Source/Reference (if necessary)**           |
-| `fh`                             | Forest canopy height (m)                    | Fused GEDI/Landsat data. Data Period=2020. ([Potapov et al., 2020](https://doi.org/10.1016/j.rse.2020.112165)) |
-| `clu`                            | Forest clumping index (dimensionless)       | GriddingMachine/MODIS. Data Period=2001-2017 Climatology. ([Wei et al., 2019](https://doi.org/10.1016/j.rse.2019.111296)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
-| `lai`                            | Leaf area index (m2/m2)                     | VIIRS-NPP. Data Period=2018-2020 Climatology. ([Myneni 2018](https://doi.org/10.5067/VIIRS/VNP15A2H.001)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
-| `ffrac`                          | Forest fraction (dimensionless)             | Based on [VIIRS GVF-NPP](https://www.star.nesdis.noaa.gov/jpss/gvf.php) and GriddingMachine/MODIS FFRAC/FVC from Terra.  Data Period=2020. ([DiMiceli et al., 2022](https://doi.org/10.5067/MODIS/MOD44B.061)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
+| `fh`                             | Forest canopy height (m)                    | Fused GEDI/Landsat data. Data Period=2020. Data frequency=Annual. ([Potapov et al., 2020](https://doi.org/10.1016/j.rse.2020.112165)) |
+| `clu`                            | Forest clumping index (dimensionless)       | GriddingMachine/MODIS. Data Period=2001-2017 Climatology. Data frequency=Monthly. ([Wei et al., 2019](https://doi.org/10.1016/j.rse.2019.111296)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
+| `lai`                            | Leaf area index (m2/m2)                     | VIIRS-NPP. Data Period=2020. Data frquency=Daily, interpolated from original 8-day product. ([Myneni 2018](https://doi.org/10.5067/VIIRS/VNP15A2H.001)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |


⬇️ Suggested change

-| `lai`                            | Leaf area index (m2/m2)                     | VIIRS-NPP. Data Period=2020. Data frquency=Daily, interpolated from original 8-day product. ([Myneni 2018](https://doi.org/10.5067/VIIRS/VNP15A2H.001)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
+| `lai`                            | Leaf area index (m2/m2)                     | VIIRS-NPP. Data Period=2020. Data frequency=Daily, interpolated from original 8-day product. ([Myneni 2018](https://doi.org/10.5067/VIIRS/VNP15A2H.001)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |


—
Reply to this email directly, view it on GitHub<https://github.com/noaa-oar-arl/canopy-app/pull/95#pullrequestreview-1681131112>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AZVW7M73R765MFDVBOKY4GDX7XHKHAVCNFSM6AAAAAA6CUPGBOVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTMOBRGEZTCMJRGI>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
